saltgui/static/stylesheets/dropdown.css

```diff
@@ -17,7 +17,6 @@ header .menu-dropdown-content {
 .run-command-button .menu-dropdown-content {
   display: none;
   position: absolute;
-  max-height: 210px;
```

Comment:
When you remove this, generated menu's can become too large. Currently the "templates" menu is generated. It is located in the command-box, just right of the words "Manual Run". Any menu is only shown when there are items in the menu. In this case that needs configuration in the `master` file. With the `max-height`, a menu with scrollbar is produced when there are too many menuitems.

Reply:
Ok. Let me then bring back `max-height` with proper size to address existing top menu. And maybe "template" could be moved to dedicated page where all templates could be easily displayed and executed. I can work on this just after dedicate job page.

Reply:
And according templates let me add (different pull request) sample templates to configuration in docker - this could help during testing and can be useful for new users to explore all SalGUI features.